tests/nlu/emulators/test_dialogflow.py

```diff
@@ -18,23 +18,15 @@ def test_dialogflow_response():
     norm = em.normalise_response_json(data)
 
     assert norm == {
-        "id": norm["id"],
-        "result": {
+        "responseId": norm["responseId"],
+        "queryResult": {
+            "resolvedQuery": data["text"],
             "action": data["intent"]["name"],
-            "actionIncomplete": False,
             "contexts": [],
```

Comment:
I think the `intent` key should also be returned, no?

Reply:
We cannot really specify any additional parameters except the intent name, but yes, I added it as well.